<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerRegionTypeModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerRegionTypeModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Input Label Map </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="InputVolumeComboBox" native="true">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>27</height>
         </size>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="renameEnabled" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="outputLabel">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Output Label Map </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="OutputVolumeComboBox" native="true">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>27</height>
         </size>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLRegionTypeNode</string>
         </stringlist>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="renameEnabled" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Regions List</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="RegionLabelsComboBox">
        <property name="maxVisibleItems">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Types List</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="TypeLabelsComboBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="regionColorLabel">
        <property name="text">
         <string>Region-Type Color Blend</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSlider" name="RegionTypeSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>500</height>
      </size>
     </property>
     <property name="text" stdset="0">
      <string>Volume Information</string>
     </property>
     <property name="checked" stdset="0">
      <bool>false</bool>
     </property>
     <property name="collapsed" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="qMRMLVolumeInfoWidget" name="MRMLVolumeInfoWidget" native="true">
        <property name="labelMapEditable" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLVolumeInfoWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLVolumeInfoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerRegionTypeModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputVolumeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerRegionTypeModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>OutputVolumeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerRegionTypeModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLVolumeInfoWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>InputVolumeComboBox</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>MRMLVolumeInfoWidget</receiver>
   <slot>setVolumeNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
